### PR TITLE
ALZ - Review of links v2

### PR DESCRIPTION
**Pull request by hgotar1**
changing the more info link for a recommendation that contains capacity planning in the content: https://learn.microsoft.com/azure/well-architected/performance-efficiency/capacity-planning

### CODE REVIEW

**Comment by brsteph** [comment]
@hgotar1 - I'm pointing this into a working branch to consolidate the three active PRs into one so we can check between them together.

File: checklists/alz_checklist.en.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
             "guid": "5d82e6df-6f61-42f2-82e2-3132d293be3d",
             "id": "A02.01",
             "severity": "Medium",
-            "link": "https://learn.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/azure-billing-microsoft-customer-agreement#design-recommendations"
```
Comment (brsteph) [inline checklists/alz_checklist.en.json:45]: I think the updated link in the working branch is better for the partner access - the multi-tenant wouldn't be for partner access.

We could add it to A01.03 instead as the main link.

File: checklists/alz_checklist.en.json [hunk 2]
```diff
@@ -336,13 +336,13 @@
...
             "subcategory": "Landing zones",
-            "text": "Configure Identity network segmentation through the use of a virtual Network and peer back to the hub. Providing authentication inside application landing zone (legacy).",
+            "text": "The identity subscription should have a virtual network that is peered to the hub virtual network in the platform subscription.",
             "waf": "Security",
             "guid": "9cf5418b-1520-4b7b-add7-88eb28f833e8",
             "id": "B04.01",
             "severity": "Medium",
-            "training": "https://learn.microsoft.com/azure/architecture/example-scenario/identity/adds-extend-domain",
-            "link": "https://learn.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/identity-access-landing-zones#managed-identities"
+            "training": " https://learn.microsoft.com/azure/cloud-adoption-framework/ready/landing-zone/design-area/identity-access-landing-zones",
```
Comment (brsteph) [inline checklists/alz_checklist.en.json:344]: 👍

File: checklists/alz_checklist.en.json [hunk 3]
```diff
@@ -495,7 +495,7 @@
             "id": "C02.11",
             "ammp": true,
             "severity": "High",
-            "training": "https://learn.microsoft.com/learn/paths/monitor-usage-performance-availability-resources-azure-monitor/",
```
Comment (brsteph) [inline checklists/alz_checklist.en.json:498]: 👍

File: checklists/alz_checklist.en.json [hunk 4]
```diff
@@ -737,7 +737,7 @@
             "guid": "de0d5973-cd4c-4d21-a088-137f5e6c4cfd",
             "id": "D02.01",
             "severity": "Medium",
-            "link": "https://learn.microsoft.com/azure/expressroute/expressroute-howto-macsec"
+            "link": "https://learn.microsoft.com/azure/cloud-adoption-framework/ready/azure-best-practices/define-network-encryption-requirements"
```
Comment (brsteph) [inline checklists/alz_checklist.en.json:740]: 👍

File: checklists/alz_checklist.en.json [hunk 5]
```diff
@@ -986,7 +986,7 @@
             "severity": "Medium",
             "graph": "resources| where type == 'microsoft.network/virtualnetworkgateways'| where properties.gatewayType =~ 'vpn' or properties.gatewayType == 'ExpressRoute'| extend SKUName = properties.sku.name, SKUTier = properties.sku.tier, Type = properties.gatewayType| extend compliant = SKUTier !in ('Basic', 'Standard')| project name, id, subscriptionId, resourceGroup, compliant",
             "training": "https://learn.microsoft.com/learn/modules/design-implement-azure-expressroute/",
-            "link": "https://learn.microsoft.com/azure/expressroute/expressroute-routing"
+            "link": "https://learn.microsoft.com/azure/expressroute/expressroute-about-virtual-network-gateways"
```
Comment (brsteph) [inline checklists/alz_checklist.en.json:989]: 👍

File: checklists/alz_checklist.en.json [hunk 6]
```diff
@@ -1245,7 +1245,7 @@
             "ammp": true,
             "severity": "High",
             "training": "https://learn.microsoft.com/learn/paths/secure-networking-infrastructure/",
-            "link": "https://learn.microsoft.com/azure/app-service/networking-features"
+            "link": "https://learn.microsoft.com/azure/cloud-adoption-framework/ready/azure-best-practices/plan-for-inbound-and-outbound-internet-connectivity"
```
Comment (brsteph) [inline checklists/alz_checklist.en.json:1248]: 👍

File: checklists/alz_checklist.en.json [hunk 7]
```diff
@@ -1518,7 +1518,7 @@
             "id": "D08.02",
             "severity": "Medium",
             "training": "https://learn.microsoft.com/learn/paths/secure-application-delivery/",
-            "link": "https://learn.microsoft.com/azure/app-service/networking-features"
+            "link": "https://learn.microsoft.com/azure/cloud-adoption-framework/ready/azure-best-practices/connectivity-to-azure-paas-services"
```
Comment (brsteph) [inline checklists/alz_checklist.en.json:1521]: 👍

File: checklists/alz_checklist.en.json (hunk 1 above)
Comment (brsteph) [reply checklists/alz_checklist.en.json:45]: @hgotar1 - This is the only one that I think needs adjustment from my side.

File: checklists/alz_checklist.en.json (hunk 2 above)
Comment (igorjnzl) [reply checklists/alz_checklist.en.json:344]: Do we need `training` URL as `link` has the same URL?

Comment (brsteph) [reply checklists/alz_checklist.en.json:344]: I think one of the "style guide" things we've talked about is that Training should always be a training module, not just a page on the topic?  But I think there are some places where there is not specific training.

But I agree we shouldn't double up on the same link.

**Comment by hgotar1** [comment]
Will proceed with a shorter pull request

**Pull request closed without merging**